library/src/main/java/com/chuckerteam/chucker/internal/ui/transaction/TransactionPayloadFragment.kt

```diff
@@ -51,7 +51,7 @@ internal class TransactionPayloadFragment :
 
     private lateinit var viewModel: TransactionViewModel
 
-    private val uiScope = CoroutineScope(Dispatchers.Main) + Job()
+    private val uiScope = MainScope() + Job()
```

Comment:
There is no need to adding `Job` to the scope. `MainScope` already uses `SupervisorJob`.

Reply:
👍